docs/contributing/coding-standards.md

```diff
@@ -94,5 +94,10 @@ Besides the rules that are checked by automatic tools, we have few rules for whi
     - Controllers
     - Facades
     - Repositories
+    - whole namespaces 
```

Comment:
But one of the acceptance criteria is "we have the rule documented" and this seems to me like proper documentation.

Reply:
You are right.. confusing situation for me. I suggest to contact our product owner because when we have a sniffer/fixer, then we don't document it in "coding standards"

Another issue is that the documentation is located in section "Standards that are not checked automatically" - and that is not true ;-)

Reply:
Yes, I didn't notice it's in that section (Standards that are not checked automatically).
It doesn't make sense this way. For now, I removed the note and I will discuss it with product owner.